### PR TITLE
change node to 12 (.nvmrc) && package-lock=false && dependencies update

### PR DESCRIPTION
**Pull request by sidoruk-sv**
* protractor `v5.4.1` -> `v7.0.0`;
* selenium-webdriver: `v2.53.0` -> `v3.6.0`;
* uuid: `v2.0.2` -> `v8.1.0`;
* lodash.merge `4.6.1` -> `4.6.2`;
* yoshi: `v3.7.0` -> `v4.103.2`.

### CODE REVIEW

**Comment by sidoruk-sv** [comment]
@yanivefraim, please review

File: .nvmrc [hunk 1]
```diff
@@ -1 +1 @@
-8
+10
```
Comment (yanivefraim) [inline .nvmrc:1]: we might bump to 12 already...

**Comment by sidoruk-sv** [comment]
Fix review notes && bumped version to v3

**Comment by sidoruk-sv** [comment]
@yanivefraim yep that was bumped in another PR since this PR was created)
https://github.com/wix/eyes.it/pull/37

So reverted major version change)

**Comment by sidoruk-sv** [comment]
@yanivefraim have no permissions to merge PRs in this repo, could you? please)

**Pull request merged**